databricks/koalas/plot/core.py

```diff
@@ -338,31 +343,42 @@ def _get_plot_backend(backend=None):
 
             KoalasPlotAccessor._backends["matplotlib"] = module
 
+        if backend == "plotly":
+            try:
+                # test if plotly can be imported
+                import plotly  # noqa: F401
+                from databricks.koalas.plot import plotly as module
+            except ImportError:
+                raise ImportError(
+                    "matplotlib is required for plotting when the "
+                    "default backend 'matplotlib' is selected."
```

Comment:
The error message sounds weird? We should replace "matplotlib" with "plotly" here?
Also "default" is not needed?

Reply:
oops